pygments/lexers/maxima.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+"""
+    pygments.lexers.maxima
+    ~~~~~~~~~~~~~~~~~~~~~~~
+
+    Lexer for the computer algebra system Maxima.
+
+    Derived from pygments/lexers/algebra.py.
+"""
```

Comment:
This is missing the license blurb. Please remove the first line indicating the coding style as well.

Reply:
I put in a license blurb and removed the encoding comment.